apps/signup/signup.js

```diff
@@ -3,18 +3,31 @@ var protoTokenUrl = "../../auth/Token/proto";
 var permissions;
 const store = new Store('../../data/');
 
-function addUser(){
```

Comment:
Why did you delete this?

Reply:
@birm My apologies for the mix-up. Yes I sometimes use AI to help to get help and check my code standards and accidentally pushed the wrong file.  I've uploaded the correct file. Could you please review it? Also updated **After result** video in description.

Reply:
I've made some changes to the message style for incorrect email entries only. Please review it and let me know if it looks good to you. If so, I'll proceed to update the code for the successful user registration message.